File: ruby/elements/css.rb
require_relative '../utils'

module Elements
  class Css < Element
    include Utils

    def initialize( compiler, article, lines)
      @lines = lines
    end

    def to_data( compiler, article)
      {'type' => 'css', 'text' => @lines.join( "\n")}
    end
  end
end

File: java/src/com/alofmethbin/jadawin/elements/Css.java
package com.alofmethbin.jadawin.elements;

import com.alofmethbin.jadawin.Article;
import java.util.List;

public class Css extends Element {
    private List<String> lines;
    
    public Css( Article article, List<String> lines) {
        super( article);
        this.lines = lines;
    }
}
